src/Lean/Elab/BuiltinTerm.lean

```diff
@@ -164,7 +164,7 @@ def elabScientificLit : TermElab := fun stx expectedType? => do
 
 @[builtinTermElab doubleQuotedName] def elabDoubleQuotedName : TermElab := fun stx _ => do
   match stx[1].isNameLit? with
-  | some val => toExpr (← resolveGlobalConstNoOverloadWithInfo stx[1] val)
+  | some val => toExpr (← resolveGlobalConstNoOverloadCore val)  -- TODO: restore info
```

Comment:
@leodemoura I think (double) quoted names should not be literals but use the regular `ident` parser. Otherwise they will not work correctly when quoted since they currently lack the preresolved information. Is there any reason not to change their parser?

Reply:
I would need a context switch to answer this question. Could you please add comments here explaining the issue? It would also be useful to open a new issue with examples that do not work.
I am assuming this PR can be merged independently of this issue. That is, it is not introducing any new issue.

Reply:
With the current state of the PR we would lose go-to-definition on double-quoted names, so we might want to resolve the `ident` issue first. Opened https://github.com/leanprover/lean4/issues/586.